pkg/kubelet/pod_workers.go

```diff
@@ -131,6 +136,7 @@ func (p *podWorkers) UpdatePod(pod *api.Pod, mirrorPod *api.Pod, updateComplete
 		// the channel is empty, so buffer of size 1 is enough.
 		podUpdates = make(chan workUpdate, 1)
 		p.podUpdates[uid] = podUpdates
+		updateType = metrics.SyncPodCreate
```

Comment:
I don't like implicitly tying kind of operation (create or update) with the lifetime of a worker corresponding to a given pod. It'll work for now, but please add a comment that you do this.